File: ARCADE/The Core/Level 4_7: increaseNumberRoundness.cs
    using System;
     
    public class Test
    {
    	public static bool increaseNumberRoundness(int n){
            while (n % 10 == 0){
                n = n / 10;
            }
            while (n % 10 != 0){
                n = n / 10;
            }
            if (n != 0) return true;
            return false;
        }
    	public static void Main()
    	{
    		int n = 902200100;
    		Console.Write(increaseNumberRoundness(n));
    	}
    }

File: ARCADE/The Core/Level 4_7: increaseNumberRoundness.pas
program increaseNumberRoundness;

function increaseNumberRoundness(n: LongInt): boolean;
begin
    while(n mod 10 = 0) do
        n := n div 10;
    
    while(n mod 10 <> 0) do
        n := n div 10;
    
    if(n <> 0) then increaseNumberRoundness := true
    else increaseNumberRoundness := false;
end;

var n : LongInt;
begin
	n := 902200100;
	writeln(increaseNumberRoundness(n));
	n := 11000;
	writeln(increaseNumberRoundness(n));
end.
